core/src/main/resources/error/error-classes.json

```diff
@@ -77,6 +111,14 @@
     "message" : [ "PARTITION clause cannot contain a non-partition column name: %s" ],
     "sqlState" : "42000"
   },
+  "NOT_MATCH_FUNCTION_NAME" : {
```

Comment:
This doesn't make much sense grammatically. How about `ADD_NEW_FUNCTION_NAME_MISMATCHED`?